core/src/main/resources/hudson/util/HudsonIsLoading/index_pt_BR.properties

```diff
@@ -21,5 +21,5 @@
 # THE SOFTWARE.
 
 Please\ wait\ while\ Jenkins\ is\ getting\ ready\ to\ work=Por favor aguarde enquanto o Jenkins se prepara para trabalhar
```

Comment:
This needs updating still (capturing as outstanding)

Reply:
Done, updated the translations that I could, removed those that I couldn't